common/datasets/librispeech.py

```diff
@@ -0,0 +1,181 @@
+"""
+This file contains helper functions for the (common) pipeline steps needed to use the LibriSpeech corpus
+"""
+import os
+
+from recipe.i6_core.audio.encoding import BlissChangeEncodingJob
+from recipe.i6_core.corpus.transform import MergeCorporaJob, MergeStrategy
+from recipe.i6_core.datasets.librispeech import *
+from recipe.i6_core.meta.system import CorpusObject
+
+
+durations = {
+    "train-clean-100": 100.6,
+    "train-clean-360": 363.6,
+    "train-other-500": 496.7,
+    "dev-clean": 5.4,
+    "dev-other": 5.3,
+    "test-clean": 5.4,
+    "test-other": 5.1,
+}
+
+durations["train-clean-460"] = (
+    durations["train-clean-100"] + durations["train-clean-360"]
+)
+durations["train-other-960"] = (
+    durations["train-clean-460"] + durations["train-other-500"]
+)
+
+
+def get_bliss_corpus_dict(audio_format="flac", create_alias_with_prefix=None):
+    """
+    Download and create a bliss corpus for each of the LibriSpeech training corpora and test sets,
+    and return all corpora as a single corpus dict.
+
+    :param str audio_format: flac (no re-encoding), wav or ogg
+    :param str|None create_alias_with_prefix:
+    :return: A corpus dict with the following entries:
+        - 'dev-clean'
+        - 'dev-other'
+        - 'test-clean'
+        - 'test-other'
+        - 'train-clean-100'
+        - 'train-clean-360'
+        - 'train-clean-460'
+        - 'train-other-500'
+        - 'train-other-960'
+    :rtype: dict[str, Path]
+    """
+
+    create_alias_with_prefix = os.path.join(create_alias_with_prefix, "LibriSpeech")
+
+    download_metadata_job = DownloadLibriSpeechMetadataJob()
+    if create_alias_with_prefix:
```

Comment:
but what if i want to create an alias without a prefix? i think this should be possible

Reply:
you can pass ""

Reply:
ah ok i thought that we always want to create `alias` dir and thus when it is None we set it to `""`. But yeah ok so with `None` it will not create the alias dir.

Reply:
we could say we always want an alias... I would always use one... but maybe there are some people who do not want this